### PR TITLE
fix #73292 pcworld.com

### LINKED ISSUE #73292: pcworld.com
### Issue URL (Annoyance)
 [https://www.pcworld.com/article/3604828/intels-debut-iris-xe-graphics-cards-are-headed-to-prebuilt-pcs-but-not-for-gaming.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.pcworld.com%2Farticle%2F3604828%2Fintels-debut-iris-xe-graphics-cards-are-headed-to-prebuilt-pcs-but-not-for-gaming.html)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Firefox
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404, <br/>NoCoin Filter List

### PR DESCRIPTION
**Pull request by stanislav-atr**
#73292

### CODE REVIEW

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -81,8 +81,8 @@ foromtb.com,dekudeals.com,encuentra24.com,krone.at##body div.fc-consent-root
...
+pcworld.com,preis.de,vorname.com,avclub.com,clickhole.com,deadspin.com,gizmodo.com,jalopnik.com,jezebel.com,kotaku.com,lifehacker.com,splinternews.com,theinventory.com,theonion.com,theroot.com,thetakeout.com,motorcyclenews.com,techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#div[id^="sp_message_"] { display: none !important; }
+pcworld.com,faz.net,chip.de,preis.de,giga.de,motorcyclenews.com,finanzen.net,vorname.com,jezebel.com,theonion.com,theinventory.com,thetakeout.com,theroot.com,avclub.com,deadspin.com,lifehacker.com,computerbild.de,techbook.de,lvz.de,sueddeutsche.de,gutefrage.net,shieldsgazette.com,spieletipps.de,ekstrabladet.dk,focus.de,demorgen.be,volkskrant.nl,trouw.nl,parool.nl,techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#html.sp-message-open { overflow: visible !important; position: static !important; margin-top: 0!important; }
```
Comment (zloyden) [inline AnnoyancesFilter/sections/cookies_specific.txt:85]: Video is broken https://www.pcworld.com/video/106661/how-to-use-windows-security-in-windows-10

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (stanislav-atr) [reply AnnoyancesFilter/sections/cookies_specific.txt:85]: Video works fine on my end, i dont see anything common with rules on this video.
<details><summary>Screenshot:</summary>

image
</details><br/>

Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_specific.txt:85]: Have you checked it under VPN? Look what I've got
<details><summary>Screenshot:</summary>

image

</details><br/>

Comment (stanislav-atr) [reply AnnoyancesFilter/sections/cookies_specific.txt:85]: Video also won't load with VPN on and AG off

Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_specific.txt:85]: After accepting cookies manually
<details><summary>Screenshot:</summary>

image

</details><br/>

Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_specific.txt:85]: @AdamWr @BlazDT please check it on your end.

Comment (AdamWr) [reply AnnoyancesFilter/sections/cookies_specific.txt:85]: On my end video works fine only if I accept cookies.

Comment (stanislav-atr) [reply AnnoyancesFilter/sections/cookies_specific.txt:85]: I've cleared my cookies and cache and video still work on my side for some reason =\
Should i remove these rules and add clicker tweak instead?

Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_specific.txt:85]: Of course.

**Review by zloyden — APPROVED** [review]

**Pull request merged**